src/app.js

```diff
@@ -1 +1,60 @@
-// write code here
+/* eslint-disable no-console */
+const fs = require('fs').promises;
+const path = require('path');
+
+const [sourcePath, destinationPath] = process.argv.slice(2);
+
+const moveFile = async () => {
+  if (!sourcePath || !destinationPath) {
+    console.error('No source or destination provided');
+
+    return;
+  }
+
+  try {
+    await fs.access(sourcePath);
+  } catch {
+    console.error("Source file doesn't exist");
+
+    return;
+  }
+
+  let destinationPathFull = destinationPath;
+
+  try {
+    const destStat = await fs.lstat(destinationPath);
+
+    if (destStat.isDirectory()) {
+      destinationPathFull = path.join(
+        destinationPath,
+        path.basename(sourcePath),
+      );
+    }
+  } catch (err) {
+    if (err.code === 'ENOENT' && destinationPath.endsWith('/')) {
+      console.error("Destination directory doesn't exist");
```

Comment:
Instead of just logging an error when the destination directory doesn't exist, consider creating the directory using `fs.mkdir(destinationPath, { recursive: true })` to ensure it exists before proceeding.